haystack/preview/components/embedders/openai_document_embedder.py

```diff
@@ -66,6 +66,16 @@ def __init__(
         if organization is not None:
             openai.organization = organization
 
+    def _get_telemetry_data(self) -> Dict[str, Any]:
+        """
+        Data that is sent to Posthog for usage analytics.
+        """
+        return {
+            "model": self.model_name,
+            "uses_prefix_or_suffix": bool(self.prefix or self.suffix),
+            "embeds_metadata_fields": bool(self.metadata_fields_to_embed),
```

Comment:
Do we need `uses_prefix_or_suffix` or `embeds_metadata_fields`? For `model` I see how it's useful but the others we don't really need. Do you maybe have an example of how this might become useful? Otherwise I would suggest we leave it out.

Reply:
Same for these two keys in the other components.

Reply:
I removed them: we can add them later if/when we actually need them :blush: